### ISSUE TITLE
Table component doesn't feature tab focus navigation

### ISSUE THREAD

**Issue by adamzerella** [alphagov/govuk-design-system#632]
It seems that the `Table` [component](https://design-system.service.gov.uk/components/table/) doesn't include the attribute `tabindex` which means it cannot be focused. Tab navigation is a common accessibility feature, I think it might be worth having the community discussion if a Table should feature Tab navigation, I believe it does.
